### ISSUE TITLE
508-defect-4 [SEMANTIC HTML]: Multi-step form headers should not have extra ARIA attributes

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#19953]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I sometimes do not hear the form heading announced properly because it is contained in a DIV with multiple ARIA attributes.

## Details

Our FormNav.jsx has 4 extra ARIA attributes [starting on line 86](https://github.com/department-of-veterans-affairs/vets-website/blob/9c610f8981da2b3608af0e4bc10519072d78d158/src/platform/forms-system/src/js/components/FormNav.jsx#L86) that could be removed. We have these same attributes in the [Segmented Progress Bar component](https://design.va.gov/storybook/?path=/story/components-progress-bars-segmentedprogressbar--default), which is where they should be. This is a small improvement discovered during that remediation effort. Screenshot and code snippet attached below.

## Acceptance Criteria

- [ ] Extra ARIA markup is removed from FormNav.jsx
- [ ] 0 new axe violations are reported in e2e and unit tests

## Proposed Solution (if known)

```diff
! FormNav.jsx#86
! https://github.com/department-of-veterans-affairs/vets-website/blob/9c610f8981da2b3608af0e4bc10519072d78d158/src/platform/forms-system/src/js/components/FormNav.jsx

<div
- aria-valuenow={current}
- aria-valuemin="1"
- aria-valuetext={stepText}
- aria-valuemax={chapters.length}
  className="nav-header nav-header-schemaform"
>
  {showHeader && (
    <h2 id="nav-form-header" className="vads-u-font-size--h4">
      {stepText}
    </h2>
  )}
  {!showHeader && (
    <div className="vads-u-font-size--h4">{stepText}</div>
  )}
</div>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

![Screen Shot 2021-02-12 at 4 27 01 PM](https://user-images.githubusercontent.com/934879/107828817-3e4cda00-6d4f-11eb-83be-f9f0dbae5e1a.png)

**Comment by taylorkaren** [comment]
Issue moved to [department-of-veterans-affairs/va-forms-system-core #254](https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/254) via [**ZenHub**](https://www.zenhub.com/)
